samples/skills/ClassificationSkill/Question/skprompt.txt

```diff
@@ -0,0 +1,22 @@
+Please decide a sentence type based on whether it is a interrogative sentence.
+
+Interrogative sentences are typically marked by inversion of the subject and predicate; that is, the first verb in a verb phrase appears before the subject. 
```

Comment:
Since you're explicitly defining a question, should we also consider the simple case of sentences ending in a question mark?

Reply:
I thought about it but ultimately left out the question mark since it may be misclassified in examples such as
For instance the sentence: This is what it has come to? is not a question though it ends with a question mark.

I do plan to use this against voice input where the intonation of the sentence could make it a question if spoken with a rising intonation.

Maybe I'm just overthinking it since I'm new to the "prompt engineering" space so open to making suggested changes :)

Reply:
Sounds good - thank you for the clarification.